packages/web-runtime/src/container/sse.ts

```diff
@@ -195,11 +196,21 @@ export const onSSEProcessingFinishedEvent = async ({
       })
     }
 
-    resourcesStore.updateResourceField({
-      id: sseData.itemid,
-      field: 'processing',
-      value: false
+    /**
+     * Resource not changed, don't fetch more data
+     */
+    if (resource.etag === sseData.etag) {
+      return resourcesStore.updateResourceField({
+        id: sseData.itemid,
+        field: 'processing',
+        value: false
+      })
+    }
+
+    const { resource: updatedResource } = await clientService.webdav.listFilesById({
```

Comment:
shouldn't this be `getFileInfo` instead of `listFilesById`?

Reply:
oh yeah, we can do this now, since we also retrieve the space id, via the sse event